typescript/phoenix-sdk/tsconfig.json

```diff
@@ -1,22 +1,18 @@
 {
-    "compilerOptions": {
-        "baseUrl": "src",
-        "module": "commonjs",
-        "target": "es2015",
-        "esModuleInterop": true,
-        "moduleResolution": "node",
-        "resolveJsonModule": true,
-        "declaration": true,
-        "paths": {
-          "@instructions/*": ["instructions/*"],
-          "@types/*": ["types/*"]
-        },
-        "outDir": "./dist"
-      },
-      "include": [
-        "src/**/*.ts",
-      ],
-      "exclude": [
-        "tests/**/*"
-      ]
-}
\ No newline at end of file
+  "compilerOptions": {
+    "baseUrl": "src",
+    "module": "commonjs",
+    "target": "es2015",
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "declaration": true,
+    "types": ["mocha"],
+    "paths": {
+      "@instructions/*": ["instructions/*"],
+      "@types/*": ["types/*"]
+    },
+    "outDir": "./dist"
+  },
+  "include": ["src/**/*.ts", "test/**/*.ts"]
```

Comment:
lol, i wonder why we had tests/ excluded

Reply:
I think we should also include examples/ unless we have a reason to not run TS compiler on those